examples/features/annotated-namespace/README.md

```diff
@@ -0,0 +1,144 @@
+# Test kernel to kernel connection with annotated namespace
```

Comment:
Please add this into 'includes'  of _features/READM.md_

Reply:
added